src/main/java/uk/gov/ons/ctp/response/collection/exercise/client/ActionSvcClient.java

```diff
@@ -22,29 +73,154 @@
    * @return ActionPlanDTO representation of the created action plan
    * @throws RestClientException for failed connection to action service
    */
-  ActionPlanDTO createActionPlan(String name, String description, HashMap<String, String> selectors)
-      throws RestClientException;
+  @Retryable(
+      value = {RestClientException.class},
+      maxAttemptsExpression = "#{${retries.maxAttempts}}",
+      backoff = @Backoff(delayExpression = "#{${retries.backoff}}"))
+  public ActionPlanDTO createActionPlan(
+      final String name, final String description, final HashMap<String, String> selectors) {
+    log.debug("Posting to action service to create action plan");
+    UriComponents uriComponents =
+        restUtility.createUriComponents(appConfig.getActionSvc().getActionPlansPath(), null);
+
+    ActionPlanDTO actionPlanDTO = new ActionPlanDTO();
+    actionPlanDTO.setName(name);
+    actionPlanDTO.setDescription(description);
+    actionPlanDTO.setCreatedBy("SYSTEM");
+    actionPlanDTO.setSelectors(selectors);
+    HttpEntity<ActionPlanDTO> httpEntity = restUtility.createHttpEntity(actionPlanDTO);
+
+    ActionPlanDTO createdActionPlan =
+        restTemplate.postForObject(uriComponents.toUri(), httpEntity, ActionPlanDTO.class);
+    log.with("action_plan_id", createdActionPlan.getId())
+        .debug("Successfully posted to action service to create action plan");
+    return createdActionPlan;
+  }
 
   /**
-   * Request list of action plans with given selectors
+   * Request list of business action plans with given selectors
    *
    * @param collectionExerciseId collectionExerciseId to find action plans for
    * @param activeEnrolment boolean for if sample unit has an active enrolment associated with it
    * @return List<ActionPlanDTO> representation of the created action plan
    */
-  List<ActionPlanDTO> getActionPlansBySelectors(
-      String collectionExerciseId, boolean activeEnrolment);
+  @Retryable(
```

Comment:
The `@Retryable` annotation should be used very sparingly. We can't have users waiting 25 seconds for a response.